adafruit_touchscreen.py

```diff
@@ -27,8 +27,13 @@
 from digitalio import DigitalInOut
 from analogio import AnalogIn
 
+try:
+    from typing import Optional, Tuple, Union
+except ImportError:
+    pass
 
-def map_range(x, in_min, in_max, out_min, out_max):
+
+def map_range(x, in_min, in_max, out_min, out_max) -> float:
```

Comment:
The input arguments should be:

```suggestion
def map_range(x: float, in_min: int, in_max: int, out_min: int, out_max: int) -> float:
```

Reply:
yes, I should have finished that - sorry